### ISSUE TITLE
use mouse wheel to switch between apps in group

### ISSUE THREAD

**Issue by samogot** [jake-phy/WindowIconList#113]
in ubuntu when i set mouse pointer at apps group and scroll mouse wheel i cyclically switch between apps in group. would be nice to have such feature here. i saw something familiar in one of forks, but using keyboard shortcuts instead of mouse wheel: https://github.com/lotabout/WindowIconList/commit/2f9c5bec1bedb60adc40261109a1043fca58cb6b

**Comment by jake-phy** [comment]
Yes good idea.  I think I will do the same as the feature request below and put this on hold till after release.
